packages/react/src/components/Modal/index.story.tsx

```diff
@@ -55,6 +55,11 @@ export default {
       </OverlayProvider>
     )
   },
+  parameters: {
+    storyshots: {
+      skip: true,
+    },
+  },
```

Comment:
I turned off the generation of `jest-styled-components` because the order of the css snapshots changes with each execution. I will turn it back on in v4.